test/integration/attachment.test.js

```diff
@@ -72,7 +72,7 @@ describeAuthOnly('attachments - authenticated', function () {
   })
 
   it('should return 201 - json array uploaded', async function () {
-    const attachment = ['test']
+    const attachment = [{ key: 'test' }]
```

Comment:
['test'] isn't valid JSON

Reply:
my bad, I thought `['test']` wasn't valid JSON after stringifying. I see it is now. The test was failing because I thought JSON arrays had to be arrays of objects so set the spec that way, all reverted.